CONTRIBUTING.md

```diff
@@ -2,11 +2,30 @@
 
 🎈 We are so happy to have you!
 
-Todo
+We're trying to create a new way of decentralized work!
+
+There are many ways to contribute, including:
+1. Get involved in the protocol design decisions [Joining Discord]() !
```

Comment:
Here's a link:

1. Get involved in the protocol design decisions by [joining our Discord](https://discord.gg/FGw97wHZJJ) !